src/brandbar.module.css

```diff
@@ -0,0 +1,13 @@
+.dropdownItem {
+  padding: 8px;
+  color: white;
+}
+
+.dropdownItem:hover {
+  color: black;
+}
```

Comment:
These colours aren't going to work with the default (openflight) branding.  Colours such as this should be specified in the `branding.css` files in landing page and picked up by the webapp.

Remove these colours and make the changes to branding.css I've asked for on flight-landing-page.